content/post/release-of-1.8.0.md

```diff
@@ -0,0 +1,419 @@
+tags: ["release"]
+draft: false
+---
+
+The time has come for another major release! We are proud to present Gitea
+1.8.0 to the world. **This release contains two new security fixes which cannot be backported to the 1.7.0 branch, so it is recommended to update to this version.**
+In this release, we merged
+[245 pull requests](https://github.com/go-gitea/gitea/pulls?utf8=%E2%9C%93&q=is:pr+milestone:1.8.0+is:merged)
+-- This is more than 1.7.0 (157) and 1.6.0 (178).
+
+You can download one of our pre-built binaries from our
+[downloads page](https://dl.gitea.io/gitea/1.8.0/) - make sure to select the
+correct platform! For further details on how to install, follow our
+[installation guide](https://docs.gitea.io/en-us/install-from-binary/).
+
+We'd like to thank all of our backers on
+[Open Collective](https://opencollective.com/gitea), who are helping us deliver
+a better piece of software.
+
+With that out of the way, here's what's new in Gitea version 1.8.0:
+
+<!--more-->
+
+## OAuth2 Provider [#5378](https://github.com/go-gitea/gitea/pull/5378)
+
+Gitea is now an OAuth2 Provider. This fixes and closes issue [#27](https://github.com/go-gitea/gitea/issues/27)!
+
+_Thanks to [**@jonasfranz**](https://github.com/jonasfranz)_
+
+## Repositories can now be archived [#5009](https://github.com/go-gitea/gitea/pull/5009)
+
+Repositories in Gitea can now be archived. This prevents the creation of new issues,
+pull-requests or pushing to the git repository.
+
+_Thanks to [**@kolaente**](https://github.com/kolaente)_
+
+## Private and Internal Organizations [#1763](https://github.com/go-gitea/gitea/pull/1763)
+
+Organizations can now be set to be Public, Internal or Private. Repositories created in
+internal organizations will only be visible to logged-in users. Private organizations will
+only be visible to those who are members.
+
+_Thanks to [**@techknowlogick**](https://github.com/techknowlogick) &amp; [**@DblK**](https://github.com/dblk)_
+
+## Changelog
+
+* SECURITY
+  * Prevent remote code execution vulnerability with mirror repo URL settings (#6593) (#6594)
+  * Resolve 2FA bypass on API (#6676) (#6674)
+  * Prevent the creation of empty sessions for non-logged in users (#6690) (#6677)
+* BREAKING
+  * Add "ghost" and "notifications" to list of reserved user names. (#6208)
+  * Change sqlite DB path default to data directory (#6198)
+  * Adds MustChangePassword to user create/edit API (#6193)
+  * Disable redirect for i18n (#5910)
+  * Releases API paging (#5831)
+  * Allow Macaron to be set to log through to gitea.log (#5667)
+  * Don't close issues via commits on non-default branch (#5622)
+* FEATURE
+  * Add regenerate secret feature for oauth2 (#6291)
+  * Expose issue stopwatch toggling via API (#5970)
+  * Add other session providers (#5963)
+  * Pull request conflict files detection (#5951)
+  * Integrate OAuth2 Provider (#5378)
+  * Implement "conversation lock" for issue comments (#5073)
+  * Feature: Archive repos (#5009)
```

Comment:
Maybe just drop the `Feature: ` here since it's already a bullet point under `Features`.

Reply:
done